src/services/development.ts

```diff
@@ -80,44 +81,29 @@ export const services: ServiceDefinition[] = [
           console.warn('Found an unhandled %s request to %s', req.method, req.url.href)
         },
       })
+      return rest
     },
     arguments: [
       $.mswHandlers,
     ],
   }],
-
-  // old style kuma mocks
-  [$.mswKumaMocks, {
-    service: (env: Env) => setupHandlers(env.var('KUMA_API_URL'), mocks, env),
+  [$.mswFakeApiHandlers, {
+    service: (env: TokenType<typeof $.env>, fs: FS) => fakeApi(env, rest, fs),
     arguments: [
-      $.Env,
+      $.env,
+      $.fakeFS,
     ],
     labels: [
       $.mswHandlers,
     ],
   }],
 
-  // new style kuma mocks
   [$.kumaFS, {
     constant: fs,
-    arguments: [
-      $.Env,
-    ],
```

Comment:
This is cruft left over from a previous change, you can't pass arguments to `constant`s